lua/nvim-treesitter/parsers.lua

```diff
@@ -2101,6 +2101,43 @@ list.supercollider = {
   maintainers = { "@madskjeldgaard" },
 }
 
+list.superhtml = {
+  install_info = {
+    url = "https://github.com/kristoff-it/superhtml",
+    files = {
+      "tree-sitter-superhtml/src/parser.c",
```

Comment:
Please use the `location` key instead; this is not the correct way to specify non-standard locations (can lead to race conditions with parallel install, for one thing).